libfreerdp/codec/rfx_bitstream.h

```diff
@@ -77,6 +77,13 @@ typedef struct _RFX_BITSTREAM RFX_BITSTREAM;
 			bs->byte_pos++; \
 		} \
 	} } while (0)
+#define rfx_bitstream_flush(bs) do { \
```

Comment:
Might be a good idea to make this a `static INLINE` function (adds some type safety)